system-tests/test/testConfigOverrides_spec.ts

```diff
@@ -10,6 +10,16 @@ const outputPath = path.join(e2ePath, 'output.json')
 describe('testConfigOverrides', () => {
   systemTests.setup()
 
+  systemTests.it('successfully runs valid suite-level-only overrides', {
+    spec: 'testConfigOverrides/valid-suite-only.js',
+    snapshot: true,
+    expectedExitCode: 0,
+    browser: 'electron',
```

Comment:
any reason we only want to run this in electron?

Reply:
I don't expect the test config overrides to be applied different per browser so I didn't think it was necessary

Reply:
Why a system test instead of a driver test?

Reply:
I wrote as a system test because this was fixing the case when the second tests in a describe block not successfully running when the configuration was resetting between specs.